games-strategy/0ad/0ad-0.0.22.recipe

```diff
@@ -0,0 +1,84 @@
+SUMMARY="O A.D. - Game"
+DESCRIPTION="A free, open-source, Real-Time Strategy (RTS) game."
+HOMEPAGE="https://play0ad.com"
+COPYRIGHT="Wildfire Games"
+LICENSE="0ADLicense"
+REVISION="1"
+SOURCE_URI="https://releases.wildfiregames.com/0ad-0.0.22-alpha-unix-build.tar.gz"
+CHECKSUM_SHA256="b50ef5aa8004bc60c542493d2c3ceff59ebdefadcd901d529c492c91503ed047"
+SOURCE_DIR="0ad-$portVersion-alpha"
+PATCHES="0ad-$portVersion-alpha.patch"
+
+ARCHITECTURES="!x86_gcc2 ?x86_64 ?x86"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	0ad$secondaryArchSuffix = $portVersion
+	app:0ad$secondaryArchSuffix = $portVersion
+"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libnspr4$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libopenal$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libSDL2$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libzlib$secondaryArchSuffix
+	0ad_data == $portVersion
+"
+
+BOOST_PKG_NAME="boost${secondaryArchSuffix}_devel"
+
+if [ "$buildArchitecture" == "x86_64" ]; then
+	BOOST_PKG_NAME="boost165_devel"
+fi
+
+BUILD_REQUIRES="
+		$BOOST_PKG_NAME
+		libxml2${secondaryArchSuffix}_devel
+		nspr${secondaryArchSuffix}_devel
+"
+
+BUILD_PREREQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cmd:gcc$secondaryArchSuffix
+	cmd:g++$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config
```

Comment:
`cmd:pkg_config` needs a `$secondaryArchSuffix`:
```diff
-	cmd:pkg_config
+	cmd:pkg_config$secondaryArchSuffix
```
Later, in `BUILD()` you should drop this whole bloc:
```
	if [ "$buildArchitecture" == "x86_gcc2" ]; then
		export PKG_CONFIG_PATH=`finddir B_SYSTEM_DEVELOP_DIRECTORY`/lib/x86/pkgconfig
	else
		export PKG_CONFIG_PATH=`finddir B_SYSTEM_DEVELOP_DIRECTORY`/lib/pkgconfig
	fi
```